### PR TITLE
remove the need for cygwin

### PR DESCRIPTION
**Pull request by jtnord**
as I keep forgetting to add cygwin onto the path before I ran the command it is better to use commands that are inbuilt into the OS

- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Review by timja — APPROVED** [review]

**Comment by jtnord** [comment]
hey Tim you are too quick :)

File: vars.cmd [hunk 1]
```diff
@@ -10,12 +10,17 @@ set JENKINS_JAVA_OPTS=-Xmx1280m
 @REM Jenkins binds to 0.0.0.0 (OMG) so we can use any network but the docker network.
 @REM but we may as well use the default network
 @echo off
-FOR /F "tokens=* USEBACKQ" %%F IN (`netsh interface ipv4 show addresses "vEthernet (WSL)" ^| grep "IP Address:" ^| gawk '{print $3}'`) DO (
+FOR /f "tokens=3" %%F in ('netsh interface ipv4 show addresses "vEthernet (WSL)" ^| findstr /c:"IP Address:"') DO (
 SET IP=%%F
 )
+IF NOT DEFINED IP (
```
Comment (jtnord) [inline vars.cmd:16]: because I always forget to start docker first - make it obvious to myself (and others!)

**Comment by jtnord** [comment]
no point waiting for a build - the script is not tested by it so shipping it to save some resources on ci.jenkins.io

**Review by timja — APPROVED** [review]

**Comment by NotMyFault** [comment]
Seems all good, build would be red anyway due to waiting for docker fixtures 🤷🏻

**Pull request merged**